pkg/catalog/routes.go

```diff
@@ -344,7 +344,11 @@ func (mc *MeshCatalog) buildAllowPolicyForSourceToDest(source *corev1.Service, d
 		Methods:   []string{constants.WildcardHTTPMethod},
 	}
 	return trafficpolicy.TrafficTarget{
-		Name:        fmt.Sprintf("%s->%s", source, destination),
+
+		// TODO(draychev): we need to improve the naming convention for allow-all routes
+		// See Github Issue: https://github.com/openservicemesh/osm/issues/1503
```

Comment:
Have a fix for this https://github.com/openservicemesh/osm/pull/1507

Reply:
It would be so nice to merge this test, then see how your change affects the test - it would be a great documentation. What do you think?
We merge https://github.com/openservicemesh/osm/pull/1504 first, then tweak https://github.com/openservicemesh/osm/pull/1507 to see how the new code works and we merge https://github.com/openservicemesh/osm/pull/1507 too?

Reply:
I don't think the order matters, I'd prefer to merge whichever can merge first and rebase/update the dependent PR.

Reply:
fix https://github.com/openservicemesh/osm/pull/1507

Reply:
Updated the code in lieu of #1507

Reply:
Thank you!